src/components/dialogs/export-dialog.jsx

```diff
@@ -177,9 +200,20 @@ const ExportDialog = ({
             onClose={handleClose}
             aria-labelledby="dialog-title-export"
         >
-            <DialogTitle>
-                {title}
-                <div style={{ marginTop: '0.8em' }} />
+            <DialogTitle>{title}</DialogTitle>
+            <DialogContent>
+                <TextField
+                    key="fileName"
+                    margin="dense"
+                    label={<FormattedMessage id="download.fileName" />}
+                    id="fileName"
+                    value={fileName}
+                    style={{ width: '100%' }}
```

Comment:
use sx instead of style, add a margin bottom 8px to insert a space to the below component

Reply:
OK